python/ray/tests/test_basic.py

```diff
@@ -2649,8 +2649,6 @@ def f():
     output_lines = captured["out"]
     for i in range(200):
         assert str(i) in output_lines
-    error_lines = captured["err"]
-    assert len(error_lines) == 0
```

Comment:
This makes me realize that this line should have been 

```python
     assert len(error_lines) == 0, error_lines
```

so that we can what the stderr was in the case of error